lib/Conversion/CalyxToFSM/CalyxToFSM.cpp

```diff
@@ -268,6 +269,99 @@ LogicalResult CompileFSMVisitor::visit(StateOp currentState, EnableOp enableOp,
   return success();
 }
 
+// CompileInvoke is used to convert invoke operations to group operations and
+// enable operations.
+class CompileInvoke {
```

Comment:
@rachitnigam   `CompileInvoke` is used to compile invoke.I iterated over all the invoke operations.

Reply:
It may be easier if we submit this in a separate pull request. This isn't necessary for an initial Invoke commit IMO.